README.md

```diff
@@ -156,4 +156,4 @@ namespace FileUploader
 
 ## Explore Further
 * [Complete Documentation](https://docs.minio.io)
-* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
\ No newline at end of file
+* [Minio .NET SDK API Reference](https://docs.minio.io/docs/.NET-client-api-reference)
```

Comment:
Are you sure this is going to be '.NET' saved as is on docs.minio.io why not keep it simple as 'dotnet-client-api-reference'